doc/languages-frameworks/rust.section.md

```diff
@@ -196,7 +196,7 @@ sometimes it may be necessary to disable this so the tests run consecutively.
 ```nix
 rustPlatform.buildRustPackage {
   /* ... */
-  cargoParallelTestThreads = false;
+  dontUseCargoParallelTests = true;
```

Comment:
Should we add that to the changelog?

Reply:
I agree. There was another API change (removal of the `target` argument of `buildRustPackage`). I can do a PR for the release notes after this one that summarizes all the changes.